### PR TITLE
KC-updates-may5

### PR DESCRIPTION
**Pull request by kcullion**
- Homepage callout updates
- new page for viewing submitted data
- new page for clinical data validation rules (@hknahal to also review and add to)
- added screenshots
- made a placeholder page for programmatic APIs

### CODE REVIEW

File: docs/data-releases.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ A DCC data release is a curated data set produced from clinical and experimental
 
 ## Data Release 1.0 
 
-**Release Date: July 30, 2020**
+**Release Date: MM/DD/YYYY**
```
Comment (rosibaj) [inline docs/data-releases.md:11]: should we do YEAR - MONTH - DAY?  That way it can be sorted easier?

File: docs/submitted-data.md [hunk 2]
```diff
@@ -2,4 +2,103 @@
...
+Check out [Clinical Data Validation Rules](clinical-data-validation-rules#clinical-data-completeness) for more information on how clinical completeness is calculated.
+
+#### % Donors with Tumour & Normal
+*Coming Soon:* Once a donor has a tumour and normal raw reed pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
+
+#### Donors in Molecular Data Processing
+*Coming Soon:* This is the number of donors that currently have samples being processed in the [analysis pipeline](dna-pipeline).
+
+#### Files to QC
+*Coming Soon:* Once an analysis workflow is completed (and the corresponding donor is clinically complete), the analysis files will be available to program members first. These files can be downloaded using the file manifest, and will be available for an embargo period before being released to the public via the File Repository (note: all molecular files are controlled and will be available for the public to download with [DACO approval](data-access)).
```
Comment (rosibaj) [inline docs/submitted-data.md:32]: - These files can be downloaded using a file manifest and the score-client

- im not sure this note is needed/adding it here is a bit confusing.  makes it sound like QC files are available for public download. [(note: all molecular files are controlled and will be available for the public to download with [DACO approval](data-access)). ]

File: docs/submitted-data.md [hunk 3]
```diff
@@ -2,4 +2,103 @@
...
+*Coming Soon:* Once a donor has a tumour and normal raw reed pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
+
+#### Donors in Molecular Data Processing
+*Coming Soon:* This is the number of donors that currently have samples being processed in the [analysis pipeline](dna-pipeline).
+
+#### Files to QC
+*Coming Soon:* Once an analysis workflow is completed (and the corresponding donor is clinically complete), the analysis files will be available to program members first. These files can be downloaded using the file manifest, and will be available for an embargo period before being released to the public via the File Repository (note: all molecular files are controlled and will be available for the public to download with [DACO approval](data-access)). 
+
+#### Donors with Released Files
+*Coming Soon:* Once the embargo period is over, a data release will occur. Donors who have BOTH core clinical completeness and analysis files available will be released to the public via the File Repository.
```
Comment (rosibaj) [inline docs/submitted-data.md:35]: files that have passed the embargo period seems more accurate

File: docs/submitted-data.md [hunk 4]
```diff
@@ -2,4 +2,103 @@
...
+
+#### All Files
+*Coming Soon:* The count of all files for the program, i.e. all clinical and molecular analysis files that have either been released to the public or are still in QC for the embargo period. A program member, having DACO approval, can download these files by using the manifest in Song. Check out [How to Download Data](data-download).
```
Comment (rosibaj) [inline docs/submitted-data.md:38]: not Song --> `using the manifest with the score-client.`

File: docs/submitted-data.md [hunk 5]
```diff
@@ -2,4 +2,103 @@
...
+
+## Donor Release Summary
+The status bar in this card shows how many program donors have fully released files (dark blue bar), partially released files (light blue bar) and no released files (white bar) out of the total committed donors.   
+![Dashbard Donor Release Summary](/assets/submission/dashboard-donor-release-card.png)  
+
+## Program Workspace Status
+The **Program Workspace** consists of the sample registration as well as the clinical submission areas.
+![Dashbard Program Workspace Status](/assets/submission/dashboard-program-workspace-card.png)  
+
+All program members share the same workspace, so members are encouraged to communicate with each other when sample registration or clinical submission is in progress.
```
Comment (rosibaj) [inline docs/submitted-data.md:48]: should this be a note? to ihghlight more?

File: docs/submitted-data.md [hunk 6]
```diff
@@ -2,4 +2,103 @@
...
+![Dashbard Program Workspace Status](/assets/submission/dashboard-program-workspace-card.png)  
+
+All program members share the same workspace, so members are encouraged to communicate with each other when sample registration or clinical submission is in progress. 
+
+**Sample Registration Progress:** this progress bar will show if there are uploaded samples that haven’t been registered yet. A blank status bar means this area is free to register samples.
+
+**Clinical Submission Progress:** this progress bar will show if a clinical submission currently has uploaded files or if it has been validated and is ready for sign off. If a submission is signed off without any updates to previously submitted data, this progress bar will become blank again, signifying that this area is free to submit clinical data.  If the progress bar is in “pending approval” state, this means the clinical submission area has been locked and is under review by DCC. Once DCC approves the clinical data updates, the progress bar will become blank and the clinical submission area is free for a new submission.
```
Comment (rosibaj) [inline docs/submitted-data.md:52]: --> under review by `the` DCC.  
--> Once the DCC approves

File: docs/submitted-data.md [hunk 7]
```diff
@@ -2,4 +2,103 @@
...
+
+## Completed Core Clinical Data
+*Coming Soon:* This line graph shows the number of donors with core clinical completeness over time. 
+
+## Molecular Data Summary
+*Coming Soon:* This line graph compares how many donors have been processed through the [DNA-Seq Pipeline](dna-pipeline): Raw Read Submission, Alignment and Sanger VC. 
+
+## Donor Data Summary
+This table is a detailed summary of the clinical data status and DNA-seq pipeline status for each registered donor in the program. It also shows which donors have files that have been released to the public via the File Repository (note: all molecular files are controlled and available for the public to download with [DACO approval](data-access))
```
Comment (rosibaj) [inline docs/submitted-data.md:61]: Again, not really sure this note about downlaod is relevent here?

File: docs/data-releases.md (hunk 1 above)
Comment (kcullion) [reply docs/data-releases.md:11]: @rosibaj I"m not sure what you mean by sorted? I was thinking of when the date is known just writing it July 12, 2020...

Comment (rosibaj) [reply docs/data-releases.md:11]: but you changed it to the mm/dd/yyyy format?

File: docs/submitted-data.md (hunk 3 above)
Comment (kcullion) [reply docs/submitted-data.md:35]: @rosibaj can we say more about the length of time for the embargo period? just wonder if program members will wonder how long that is

File: docs/data-releases.md (hunk 1 above)
Comment (kcullion) [reply docs/data-releases.md:11]: ya sorry I just changed that so we don't forget to change it from my fake date...any format is fine probably

File: docs/submitted-data.md (hunk 3 above)
Comment (kcullion) [reply docs/submitted-data.md:35]: Ok - I'll leave that for you to explain :) Not sure the details of it

File: docs/submitted-data.md [hunk 8]
```diff
@@ -2,4 +2,103 @@
...
+Once samples are registered, each corresponding donor will be counted as a percentage out of the number of committed program donors.
+
+#### % Donors with all Core Clinical Data
+Once all core clinical fields and files have been submitted for a donor, they are considered **"clinically complete"**. 
+> **A donor must be clinically complete before any of their molecular analysis files are released to the program members.**
+
+Check out [Clinical Data Validation Rules](clinical-data-validation-rules#clinical-data-completeness) for more information on how clinical completeness is calculated.
+
+#### % Donors with Tumour & Normal
+*Coming Soon:* Once a donor has a tumour and normal raw reed pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
```
Comment (hknahal) [inline docs/submitted-data.md:26]: small typo ("reed")

Comment (kcullion) [reply docs/submitted-data.md:26]: oh yea..it's "read" right?

File: docs/data-releases.md (hunk 1 above)
Comment (rosibaj) [reply docs/data-releases.md:11]: ahh i see ok!

Comment (kcullion) [reply docs/data-releases.md:11]: I'll remove the dates for now so it's not confusing

File: docs/submitted-data.md (hunk 8 above)
Comment (hknahal) [reply docs/submitted-data.md]: yes, it's "read"

**Pull request merged**